Tella/Scenes/HomeView/Views/FileListView/Add files/PhotoVideoViewModel.swift

```diff
@@ -16,17 +16,18 @@ class PhotoVideoViewModel : ObservableObject {
     var mainAppModel : MainAppModel
     var folderPathArray: [VaultFile] = []
     var  resultFile : Binding<[VaultFile]?>?
-    
+    var  rootFile : Binding<VaultFile?>?
+
     init(mainAppModel: MainAppModel,
          folderPathArray: [VaultFile],
-         resultFile : Binding<[VaultFile]?>? ) {
-        
+         resultFile : Binding<[VaultFile]?>?,
+         rootFile : Binding<VaultFile?>?) {
```

Comment:
Change Binding<VaultFile?>? to  Binding<VaultFile?> to remove more optional chaining ?
 I do not think the value of `rootfile` binding will be nil while initializing the view model.

Reply:
maybe because the root in `VaultManager` is optional that's why I put it optional, but I can use `guard let` 
anyway it will be update in merge vault file to database